### PR TITLE
Some testcases and code for them 

### PR DESCRIPTION
**Pull request by mrraghur**

### CODE REVIEW

File: May2022/2.txt [hunk 1]
```diff
@@ -0,0 +1,14 @@
+3
+1
+5
```
Comment (mrraghur) [inline May2022/2.txt:3]: Add a readme in may2022 directory.

File: May2022/Implement strStr()/main.py [hunk 2]
```diff
@@ -0,0 +1,79 @@
+from typing import List
```
[the remark this answers is not in the document]
Comment (Prashanthveddula) [reply May2022/Implement strStr()/main.py]: Names are changed

File: May2022/LengthofLastWord/testcases/Llw3.txt [hunk 3]
```diff
@@ -0,0 +1 @@
+You go and understand the tree
```
Comment (mrraghur) [inline May2022/LengthofLastWord/testcases/Llw3.txt:1]: These strings are very small. We need some with 10k+ words or even 100K+ words. We really want to stress the system. a

Comment (Prashanthveddula) [reply May2022/LengthofLastWord/testcases/Llw3.txt:1]: Okay I will add those.

File: May2022/ImplementStrstr/main.py [hunk 4]
```diff
@@ -0,0 +1,79 @@
+from typing import List
+with open('is1.txt') as f:
+    content =f.readlines()
+    l1=[x.strip() for x in content]
+    #print(l1)
+with open('is2.txt') as f:
+    content =f.readlines()
+    l2=[x.strip() for x in content]
+with open('is3.txt') as f:
```
Comment (mrraghur) [inline May2022/ImplementStrstr/main.py:9]: Please fix these. We need a for loop

**Comment by mrraghur** [comment]
ImplementStrstr : 
=============
Nothing is present.
LengthofLastWord : 
=============
              Nothing is working. 
                   No information on what failed in each testcase. 
                   No information on how many testcases failed and how many passed. 
                   Not using for loop while reading testcases.  Lot of repetitive code. 
                   Very small testcases. 
LongestCommonPrefix :
================
                 Same problems as above.
MaxSubArray :
==========
        Same problems as above.
RemoveDuplicates
=============
        Only two tests?
        Nothing is working.
        No information on what failed in each testcase.  
        No information on how many testcases failed and how many passed. 
        Not using for loop while reading testcases.  Lot of repetitive code. 
        Very small testcases. 
RemoveElement
=============
        I think we need to add comments explaining how testcases work.
        Same as above, nothing works.
RomantoInteger
=============
      This is a fairly well return code. It uses for loop, and has multiple examples.
       But when a testcase fails, it does not print what is wrong about the testcase. Only Pass/Fail info.
       Please enhance it. 
SearchInsertPosition
===============
      Needs comments on why it is reading from two files.
      Why only one testcase in code? Where is the for loop?
testcases_util.py
TwoSum
=======
    Only one testcase in code?
    Same comments as before.
    Add comments on use model.
ValidParentheses
=============
        Only two tests?
        Nothing is working.
        No information on what failed in each testcase.
        No information on how many testcases failed and how many passed.
        Not using for loop while reading testcases.  Lot of repetitive code.
        Very small testcases.

**Pull request merged**